### PR TITLE
Fixed the title of the Open source licenses page under About section in right way.

### PR DESCRIPTION
**Pull request by DreamyPhobic**
Fixes #2496 and #2397

#### What has been done to verify that this works as intended?
I tested it by opening the Open Source Licenses page under the About section. It is showing the title as intended.

#### Why is this the best possible solution? Were any other approaches considered?
This approach not only solve the issue but also allows us to reuse the WebViewActivity to display other webpages like ODK website, ODK forum etc. 

#### Are there any risks to merging this code? If so, what are they?
No.

#### Do we need any specific form for testing your changes? If so, please attach one.
No.
#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/opendatakit/docs/issues/new) and include the link below.
No.

**Thanks to @grzesiek2010 for pointing out the problem in previous solution #2496**

#### Before submitting this PR, please make sure you have:
- [x] run `./gradlew pmd checkstyle lintDebug spotbugsDebug` and confirmed all checks still pass.
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [x] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/opendatakit/collect/blob/master/CONTRIBUTING.md#ui-components-style-guidelines)

### CODE REVIEW

**Comment by lognaturel** [comment]
Yes, this does make more sense, thank you @grzesiek2010.

Please revert changes to `gradle/wrapper/gradle-wrapper.properties`, @DreamyPhobic and we can go through the regular verification process. I don't think it actually changes the behavior but it is a good improvement.

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,7 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+android.enableD8.desugaring=true
 distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
```
Comment (lognaturel) [inline gradle/wrapper/gradle-wrapper.properties]: Please make sure this also goes back to `bin`!

Comment (DreamyPhobic) [reply gradle/wrapper/gradle-wrapper.properties]: Sorry, i didn't get what you said...

Comment (lognaturel) [reply gradle/wrapper/gradle-wrapper.properties]: This file must be reverted to look exactly like https://github.com/opendatakit/collect/blob/master/gradle/wrapper/gradle-wrapper.properties

In particular, notice that the one on `master` only loads binary files for gradle. The change made here set it to `all` which also includes documentation and other files.

There should also not be a timestamp change. The easiest is probably to copy the file I linked to from master and paste that into your local copy of the file.

**Comment by kkrawczyk123** [comment]
The title has been changed, but the old one is flashing for the moment when clicking on Open source libraries/ licenses page before a new one will appear, git attached:
license

**Comment by kkrawczyk123** [comment]
@opendatakit-bot unlabel "needs testing"

**Comment by DreamyPhobic** [comment]
Actually, i am changing the title in OnPageFinished method, if showing the uri at the time loading is not fine then i can change the title in OnPageStarted method itself.

**Comment by lognaturel** [comment]
Thanks, @kkrawczyk123! This seems like normal browser behavior to me. That said, @DreamyPhobic is it actually possible to pull the title from `OnPageStarted`? I would expect that the title wouldn't be loaded by then yet. If it is possible then it might be a reasonable approach.

**Comment by DreamyPhobic** [comment]
@lognaturel  Yes you are right,  the title is not loaded by that time so it's showing blank space there. Therefore, I think it is fine to show url initially.

**Comment by lognaturel** [comment]
Thanks, @DreamyPhobic, that makes sense! I'll wait until Monday to see if @kkrawczyk123 or @grzesiek2010 has more to add.

**Review by grzesiek2010 — APPROVED** [review]

**Comment by lognaturel** [comment]
Thanks for the check, @grzesiek2010 and for making the change, @DreamyPhobic. Like @yanokwa likes to say:

team-work-makes-the-dream-work

**Pull request merged**